Component Plug-in (CP) Examples/mapField/mapField/v1/index.html

```diff
@@ -39,22 +42,34 @@
     });
 
     function initMap() {
+      Appian.Component.setValidations('');
       mapState.scriptLoaded = true;
-      changeLocation(newLocation)
+      changeLocation(newLocation);
     }
 
+    function gm_authFailure() {
```

Comment:
I think this can just be called `authFailture` - the prefix makes sense but is a bit confusing on first read

Reply:
I agree that the prefix is confusing at first glance. This function name is something their callback flow expects: https://developers.google.com/maps/documentation/javascript/events#auth-errors

I'll add a comment to explain and point to the docs.

Reply:
Ah that makes more sense